playwright-e2e/tests/dsm/participant-list/quick-filter-under-age.spec.ts

```diff
@@ -24,10 +26,15 @@ test.describe('Participants Search', () => {
       assertTableHeaders(actualHeaderNames, orderedHeaderNames);
 
       await participantsTable.sort('Date of Majority', SortOrder.DESC);
-      const dateOfMajoritySample = await participantsTable.getRowText(0, 'Date of Majority');
-      const dateOfMajority = new Date(dateOfMajoritySample);
-      const today = new Date();
-      expect(dateOfMajority > today, `Date of Majority "${dateOfMajority}" is not greater than today's date "${today}"`).toBeTruthy();
+
+      // Randomize rows
+      const rowCount = await participantsTable.getRowsCount();
+      const rowIndex = shuffle([...Array(rowCount).keys()])[0];
+
+      const dateOfMajoritySample = await participantsTable.getRowText(rowIndex, 'Date of Majority');
+      const dateOfMajority = getDate(new Date(dateOfMajoritySample));
+      const today = getDate();
+      expect(dateOfMajority >= today, `Date of Majority "${dateOfMajority}" should be later than today's date "${today}"`).toBeTruthy();
```

Comment:
Format dates to `mm/dd/yyyy`. `HH:MM:SS` substring is causing date comparison error.

```
Error: Date of Majority "Wed Jan 17 2024 00:00:00 GMT-0500 (Eastern Standard Time)" is not greater than today's date "Wed Jan 17 2024 11:23:12 GMT-0500 (Eastern Standard Time)"

    expect(received).toBeTruthy()

    Received: false

      28 |       const dateOfMajority = new Date(dateOfMajoritySample);
      29 |       const today = new Date();
    > 30 |       expect(dateOfMajority > today, `Date of Majority "${dateOfMajority}" is not greater than today's date "${today}"`).toBeTruthy();
         |                                                                                                                          ^
      31 |     });
      32 |   }
      33 | });

        at /home/circleci/repo/playwright-e2e/tests/dsm/participant-list/quick-filter-under-age.spec.ts:30:122
```